assets/css/main.css

```diff
@@ -0,0 +1,15 @@
+/* Base components */
+@import "tailwind.css";
```

Comment:
Is there any value in creating another file specially for importing Tailwind's CSS modules? I feel this could lead to some confusion.

Reply:
Not really - if it works without a separate file then we should do that... I had an issue combining them into the same file, will try this later.